src/components/core/tabs/tab.tsx

```diff
@@ -0,0 +1,27 @@
+import classNames from 'classnames';
+import css from './tab.module.scss';
+
+export interface TabProps {
+    active?: boolean;
+    active_tab_ref?: any;
+    contained?: boolean;
+    dark?: boolean;
+    icon?: string;
+    label?: string;
+    onClick?: () => void;
+}
+
+const Tab = ({ active, active_tab_ref, contained, dark, icon, label, onClick }: TabProps) => {
```

Comment:
```suggestion
const Tab = ({ active, contained, dark, icon, label, ...props }: TabProps) => {
```
We can actually use the built-in `ref` and `onClick` definitions instead of redeclaring our own